ddtrace/opentracer/tracer.py

```diff
@@ -366,11 +366,11 @@ def extract(self, format, carrier):  # noqa: A002
         self._dd_tracer.context_provider.activate(dd_span_ctx)
         return ot_span_ctx
 
-    def get_log_correlation_context(self):
-        # type: () -> Dict[str, str]
+    def get_log_correlation_context(self, active=None):
```

Comment:
I don't think we need this argument (at least not yet). 

```suggestion
    def get_log_correlation_context(self):
```

Reply:
removed